students/Dustin_L/lesson08/circle.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+"""Circle Module"""
+
+import math
+
+
+class Circle:
+    """Circle Class"""
+    def __init__(self, radius):
+        if radius < 0:
+            raise ValueError
```

Comment:
good idea to supply error message for user of the class 
`raise ValueError("value has to be greater than 0")`